cheetah/accelerator/screen.py

```diff
@@ -203,7 +196,9 @@ def reading(self) -> torch.Tensor:
         read_beam = self.get_read_beam()
         if read_beam is Beam.empty or read_beam is None:
             image = torch.zeros(
-                (int(self.effective_resolution[1]), int(self.effective_resolution[0]))
+                (int(self.effective_resolution[1]), int(self.effective_resolution[0])),
```

Comment:
```suggestion
                (self.effective_resolution[1], self.effective_resolution[0]),
```

The `int()` is unnecessary if the property satisfies its type hint.